test.sh

```diff
@@ -1,3 +1,17 @@
+#!/bin/bash
+
+NO_CLEAN=false
+REGRESS_PGSpider=false
```

Comment:
What is the purpose of "REGRESS_PGSpider"?
It seems unnecessary.

Reply:
It's unnecessary. I updated new version.

Reply:
Thank you for fixing.